.changeset/smart-vans-greet.md

```diff
@@ -0,0 +1,6 @@
+---
+"@neo4j/graphql": major
+"@neo4j/graphql-ogm": major
+---
+
+Validation of type definitions is now configured using the `validate` boolean option in the constructor, which defaults to `true`.
```

Comment:
Is it worth also mentioning the changes to custom resolver validation?

Reply:
It did get released in dev, it just defaulted to not running the validation so it wasn't a breaking change